pkg/controller/mocktests/secretmanager_secret_test.go

```diff
@@ -35,6 +31,10 @@ import (
 
 	"github.com/GoogleCloudPlatform/k8s-config-connector/mockgcp"
 	"github.com/GoogleCloudPlatform/k8s-config-connector/mockgcp/pkg/storage"
+	"github.com/GoogleCloudPlatform/k8s-config-connector/pkg/dcl/clientconfig"
+	"github.com/GoogleCloudPlatform/k8s-config-connector/pkg/test"
+	testreconciler "github.com/GoogleCloudPlatform/k8s-config-connector/pkg/test/controller/reconciler"
+	tfprovider "github.com/GoogleCloudPlatform/k8s-config-connector/pkg/tf/provider"
```

Comment:
nit: turning on the `gci` linter will help us w these down the road; you don't have to, but you could revert changes to this file.

Reply:
Yes `gci` will sort them for us but it may not follow necessarily any predefined patterns we want to enforce.